airflow/contrib/hooks/qubole_hook.py

```diff
@@ -92,6 +92,8 @@ def build_command_args():
     return command_args, list(hyphen_args)
 
 
+COMMAND_ARGS = None
+HYPHEN_ARGS = None
```

Comment:
What's this one for/about?

Reply:
Auto Api uses static code analysis to detect code structure. It can not correctly detect automatically generated variables. Setting the default value for a variable allows correct detection of this variable.